website/app/styles/markdown/typography.scss

```diff
@@ -3,7 +3,8 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-@use "../typography/mixins";
+ @use "../typography" as *;
```

Comment:
question: I suspect we don't need this line (L6), as we import the mixins below (L7) and we don't seem to need anything else from typography in this file

Reply:
Thanks, I'll double check if it's needed

Reply:
Wasn't needed so I'm removing